BUFR_TableD_en_40.csv

```diff
@@ -793,3 +793,169 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
 40,Additional satellite report sequences,340017,(Sentinel-3 (S3) level 2 water product),,013013,Total snow depth,,,Operational
 40,Additional satellite report sequences,340017,(Sentinel-3 (S3) level 2 water product),,025112,Band specific altimeter data quality flag,,,Operational
 40,Additional satellite report sequences,340017,(Sentinel-3 (S3) level 2 water product),,033092,Band specific ocean quality flag,,,Operational
+40,Additional satellite report sequences,340018,(Altimeter product),,340193,Uuu,,,Operational
```

Comment:
From here to line 799: Please replace the names with those in the proposal:


Table references | Element name
-- | --
3-40-019 | Satellite general values
3-40-020 | General radiometer values
3-40-021 | Altimeter model values
3-40-022 | Altimeter main values

Reply:
The numbers also need to be updated (i.e. 340192 has become 340019 - all rows need to be renamed).